streamparse/storm/component.py

```diff
@@ -204,13 +205,13 @@ def _setup_component(self, storm_conf, context):
                                             1000000)  # 1 MB
             backup_count = self.storm_conf.get('streamparse.log.backup_count',
                                                10)
-            log_file = ('{log_path}/streamparse_{topology_name}_{component_name}'
-                        '_{task_id}_{pid}.log'
-                        .format(log_path=log_path,
-                                topology_name=self.topology_name,
-                                component_name=self.component_name,
-                                task_id=self.task_id,
-                                pid=self.pid))
+            log_file = join(log_path,
+                            ('streamparse_{topology_name}_{component_name}'
+                             '_{task_id}_{pid}.log'
+                             .format(topology_name=self.topology_name,
+                                     component_name=self.component_name,
+                                     task_id=self.task_id,
+                                     pid=self.pid)))
```

Comment:
Big +1 to using `os.path.join`, but please double-check with @msukmanowsky. I remember having a conversation with him at one point about why we needed string formatting instead. I don't remember what it was, and I might be misremembering it as an `invoke` thing, but it's worth checking.